public/js/graph/react_graph.js

```diff
@@ -331,11 +364,28 @@ var Graph = React.createClass({
                     edgesJSON={this.state.edgesJSON}
                     svg={this}/>
                 <EdgeGroup svg={this} ref='edges' edgesJSON={this.state.edgesJSON}/>
-            </svg>
+            </svg></div>
+
         );
     }
 });
 
+var Button = React.createClass({
+
+    handleMouseDown: function() {
+        this.props.mouseDown(this.props.mouseDownArg);
+    },
+
+    render: function() {
+
+        return (
+         <div id={this.props.id}>
+        <img id={this.props.id} onMouseDown={this.handleMouseDown}
```

Comment:
You should never duplicate ids. So you'll need to figure out which one you want to have the id, I'm assuming for styling purposes.